tests/test-function-8-exp.out

```diff
@@ -0,0 +1 @@
+Evan
```

Comment:
looks like this is the wrong output that will have to be fixed before we merge this PR @evantarrh  ;)